website/pages/docs/upgrading/index.mdx

```diff
@@ -103,3 +103,13 @@ Upgrading installations of Vault which participate in [Enterprise Replication](/
 - Verify functionality of each secondary instance after upgrading
 - When satisfied with functionality of upgraded secondary instances, upgrade
   the primary instance
+
+For Vault Enterprise customers who have both Disaster Recovery (DR) and Performance Replication (PR) clusters, the recommended sequence for upgrading is as follows:
+
+* Upgrade Disaster Recovery clusters
```

Comment:
It seems weird to not have this broken out into secondary & primary the way performance replication is.

Reply:
I considered the pre-existing section immediately above my additions to address this, but let me know if you would prefer it to be emphasized again:

> - **Upgrade the replication secondary instances first** using appropriate
  guidance from the previous sections depending on whether each secondary
  instance is Non-HA or HA
> - Verify functionality of each secondary instance after upgrading
> - When satisfied with functionality of upgraded secondary instances, upgrade
  the primary instance